src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/IDocumentContextFactory.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 
-namespace Microsoft.AspNetCore.Razor.LanguageServer;
+namespace Microsoft.CodeAnalysis.Razor.Workspaces.ProjectSystem;
```

Comment:
Consider changing this namespace to `Microsoft.CodeAnalysis.Razor.ProjectSystem` to match other types in this folder. I've been doing that with all of them as I've made changes.

Reply:
Looks like I also violated that on a previous PR so I'll fix them all up. I'll also change `Microsoft.CodeAnalysis.Razor.Workspaces.DocumentMapping` to `Microsoft.CodeAnalysis.Razor.DocumentMapping` to match, and update everything. At least there is an easy code action for these sorts of updates :)

Reply:
> At least there is an easy code action for these sorts of updates

For sure! Move to Namespace is a huge time saver. I feel less bad recommending changes like these knowing that's there.